<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="verticalWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="addMovieButton">
           <property name="styleSheet">
            <string notr="true">QToolButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
QToolButton:pressed{border:4px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}</string>
           </property>
           <property name="text">
            <string>添加视频</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icon/add</normaloff>:/icon/add</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeMovieButton">
           <property name="styleSheet">
            <string notr="true">QToolButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
QToolButton:pressed{border:4px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
</string>
           </property>
           <property name="text">
            <string>移除视频</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icon/remove</normaloff>:/icon/remove</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="deleteAllButton">
           <property name="styleSheet">
            <string notr="true">QToolButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
QToolButton:pressed{border:4px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
</string>
           </property>
           <property name="text">
            <string>清空列表</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icon/delete</normaloff>:/icon/delete</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="editMovieButton">
           <property name="styleSheet">
            <string notr="true">QToolButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
QToolButton:pressed{border:4px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
</string>
           </property>
           <property name="text">
            <string>     编 辑     </string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icon/edit</normaloff>:/icon/edit</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="editProjectionButton">
           <property name="styleSheet">
            <string notr="true">QToolButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
QToolButton:pressed{border:4px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
</string>
           </property>
           <property name="text">
            <string>全景投影</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icon/VR</normaloff>:/icon/VR</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableWidget"/>
         </item>
         <item>
          <widget class="QWidget" name="verticalWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>420</width>
             <height>300</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>420</width>
             <height>300</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout" name="dshowLayout"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>输出格式</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="formatBox"/>
             </item>
             <item>
              <widget class="QPushButton" name="settingButton">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);}
QPushButton:hover{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);}
QPushButton:pressed{border:4px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);}</string>
               </property>
               <property name="text">
                <string>设置</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>125</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>输出路径</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="pathBox"/>
             </item>
             <item>
              <widget class="QPushButton" name="selectpathButton">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);}
QPushButton:hover{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);}
QPushButton:pressed{border:4px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);}</string>
               </property>
               <property name="text">
                <string>选择目录</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="openpathButton">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);}
QPushButton:hover{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);}
QPushButton:pressed{border:4px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);}</string>
               </property>
               <property name="text">
                <string>打开目录</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="stratTransformButton">
           <property name="styleSheet">
            <string notr="true">QToolButton{border:2px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
QToolButton:pressed{border:4px groove gray;border-radius:10px;padding:2px 4px;background-color: rgb(230, 230, 230);font: 75 11pt &quot;Agency FB&quot;;}
</string>
           </property>
           <property name="text">
            <string>开始转换</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icon/start</normaloff>:/icon/start</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>转换进度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="styleSheet">
            <string notr="true">QProgressBar{
        border: none;
        color: white;
        text-align: center;
        background: rgb(68, 69, 73);
}
QProgressBar::chunk {
        border: none;
        background: rgb(0, 160, 230);
}</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
